types/rpc-service.ts

```diff
@@ -78,22 +80,9 @@ export class RPCService {
 
     if (fastest.success) {
       latencies[`${networkId}__${fastest.rpcUrl}`] = fastest.duration;
```

Comment:
I wonder why two underscores.

Reply:
I have no idea, not of my making, but my guess would be to avoid issues if the url has underscores probably?